src/routes/(infoPages)/[topTierPage]/[...serviceGroupPage]/+page.server.ts

```diff
@@ -185,46 +213,89 @@ export const load = async ({
       spaceID: CONTENTFUL_SPACE_ID,
       token: CONTENTFUL_DELIVERY_API_TOKEN,
     });
-    const data = await client.fetch<ServiceGroupCollectionQuery>(printQuery(query), {
+
+    const baseData = await client.fetch<ServiceGroupQuery>(printQuery(baseQuery), {
       variables: { metadataID },
     });
-    if (!data) break fetchData;
-    const [serviceGroup] = data?.serviceGroupCollection?.items ?? [];
+    if (!baseData) break fetchData;
+    const [serviceGroup] = baseData?.serviceGroupCollection?.items ?? [];
     if (!serviceGroup) break fetchData;
     const heroImageURL = serviceGroup?.heroImage?.imageSource?.url;
     const heroImageBlurhashPromise = heroImageURL && getBlurhash(heroImageURL, { fetch });
     const descriptionBlurhashesPromise = getBlurhashMapFromRichText(serviceGroup?.description, {
       fetch,
     });
-    const childServiceEntriesPromises =
+
+    const childServiceEntryIDs =
       serviceGroup.serviceEntriesCollection?.items
-        .filter((item): item is ChildServiceEntry => item?.__typename === "ServiceEntry")
-        .map(async (entry) => ({
-          ...entry,
-          description: entry.description
-            ? {
-                ...entry.description,
-                blurhashes: await getBlurhashMapFromRichText(entry?.description, { fetch }),
-              }
-            : undefined,
-        })) ?? [];
-    const childServiceGroups =
+        ?.filter((item): item is ChildServiceEntryStub => item?.__typename === "ServiceEntry")
+        ?.map(({ sys: { id } }) => id) ?? [];
+
+    const childServiceEntryIDChunks = chunks(childServiceEntryIDs, 10);
+
+    const childServiceGroupIDs =
       serviceGroup.serviceEntriesCollection?.items
-        .filter((item): item is ChildServiceGroup => item?.__typename === "ServiceGroup")
-        .map((group) => {
-          const { id } = group?.pageMetadata?.sys ?? {};
-          if (!id) return group;
-          const { url } = pageMetadataMap.get(id) ?? {};
-          return { ...group, url };
-        }) ?? [];
+        ?.filter((item): item is ChildServiceGroupStub => item?.__typename === "ServiceEntry")
```

Comment:
Looks like the cards aren't showing up on Core Content pages; I think this needs to be:
```suggestion
        ?.filter((item): item is ChildServiceGroupStub => item?.__typename === "ServiceGroup")
```